cypress/integration/pages/liveRadio/testsForCanonicalOnly.js

```diff
@@ -44,6 +46,50 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
         });
       }
     });
+    describe('Radio Schedule', () => {
+      it('should be displayed if there is enough schedule data', () => {
+        const isRadioScheduleOnPage = serviceHasRadioSchedule({
+          service,
+          variant,
+        });
+        cy.log(
+          `Live Radio Page configured for Radio Schedule? ${isRadioScheduleOnPage}`,
+        );
+
+        if (isRadioScheduleOnPage) {
+          let schedulePath = Cypress.env('currentPath').replace(
+            'liveradio',
+            'schedule.json',
+          );
+
+          if (schedulePath.includes('bbc_afaanoromoo_radio')) {
+            // the schedule call for afaanoromoo is made to bbc_oromo_radio
+            schedulePath = schedulePath.replace(
+              'bbc_afaanoromoo_radio',
+              'bbc_oromo_radio',
+            );
+          }
```

Comment:
The `if` seems unnecessary here - `replace` is naturally not going to replace a substring if the string doesn't contain it

```suggestion
          const schedulePath = Cypress.env('currentPath').replace(
            'liveradio',
            'schedule.json',
          ).replace(
            'bbc_afaanoromoo_radio',
            'bbc_oromo_radio',
          );
```
(feel free to ignore - probably not even worth the rebuild :P)